lib/icingadb/redisconnection.hpp

```diff
@@ -53,7 +55,33 @@ namespace icinga
 	public:
 		DECLARE_PTR_TYPEDEFS(RedisConnection);
 
-		typedef std::vector<String> Query;
+		class QueryArg
```

Comment:
Please add some docs describing what this represents and how it's supposed to be used.